app/assets/stylesheets/frontend/global/_aspect-ratio.scss

```diff
@@ -0,0 +1,20 @@
+// This class should be used on an element that wraps an image to prevent jank
+// (aka that thing where the page jumps around because the image has loaded and
+// now takes up space). For much more information (and a couple of other
+// techniques) take a look at https://css-tricks.com/aspect-ratio-boxes/.
+.aspect-ratio-3\:2 {
+  background-color: govuk-colour("mid-grey", #f3f2f1);
```

Comment:
Query - why a grey background? Most of (all?) our backgrounds are white, so if an image fails to load will the alt text have enough contrast?

Reply:
It's a grey background because I was making sure that the image took up the correct amount of space (since images weren't working on the dev environment), and mistakenly left it in. Good catch - thanks!